src/app/config/index.js

```diff
@@ -0,0 +1,8 @@
+import header from './header';
+
+const appConfig =  {
```

Comment:
It's only really a style I would use for the non complex exports. But you are allowed to discuss, don't have to do everything my way ;)